### DISCUSSION TITLE
ストックへ残す際の確認モーダルをもう少し大きくしてほしい

### DISCUSSION THREAD

**Discussion by fooramu** [increments/qiita-discussions#27] [要望・アイデア・不具合]
（Categories to Stock モーダルでの）ファーストビューで表示されるカテゴリが５件ほどしかなく10件程みれるようにして頂きたいです。
ストックしようとしている記事が「どのカテゴリに属するべきかを」カテゴリ10件程を同時にみながら判断したい。

※ 10件に拘りはなく15件でも20件でも構わないです。5件だと少ないなという内容です。
※ 確認ブラウザ Chrome

**Comment by kskwtnk** [comment]
@fooramu 

ありがとうございます！
こちらの意見は他のユーザーさまからもいただいており、我々も認識しています……！

表示領域の問題含め、ストック体験を良くしようというIssueは実は立っているので改善していきます！

また、既にどんなIssueがあるか、ロードマップはどうなっているか、といった共有も今後はしていきたいと思っています！
https://github.com/qiita-discussions/qiita-discussions/discussions/19

**Reply by uhooi** [reply]
横から失礼します。
ロードマップの共有いいですね！
納期に追われてウォーターフォール的な開発になると運営側が大変なので、無理のない範囲で公開していただけると嬉しいです✨

**Reply by kskwtnk** [reply]
@uhooi 

ありがとうございます！
いつもuhooiさん優しいコメントをしてくださって……とても嬉しいです！ ☺️
